source/27-Daniel.usfm.db

```diff
@@ -397,7 +397,7 @@
 \v 15 As for me, Daniel, my spirit was grieved by reason of this, and the visions of my head troubled me.
 \v 16 I came near to one of those who stood by, and asked him the truth concerning all this. So he told me the interpretation of the things:
 \v 17 “These four great beasts are four kings who will arise out of the earth.
-\v 18 But the holy ones of the Most High will receive the sovereignty, and possess the sovereignty for ever, for ever and ever.
+\v 18 But the holy ones of the Most High will receive the sovereignty, and possess the sovereignty [us:forever|cth:for ever], [us:forever|cth:for ever] and ever.
```

Comment:
I think the second "for ever" here should remain split given the fact that it is part of the phrase "for ever and ever".